editor/src/templates/editor-canvas.tsx

```diff
@@ -150,7 +150,7 @@ function getDefaultCursorForMode(mode: Mode): CSSCursor {
     case 'live':
       return CSSCursor.BrowserAuto
     case 'textEdit':
-      return CSSCursor.Insert
+      return CSSCursor.Select
```

Comment:
mmmh I'm not super sure about this one: having the cursor change from outside a text editable area (a crosshair) from inside (a pointer) feels weird to me 🤔

Reply:
This is exactly how figma works, I think it is correct.

Figma:
![16-42-dtylt-pdnh1](https://user-images.githubusercontent.com/127662/212728756-e20bb48c-1e01-4368-afaa-4585e4766e2b.gif)

Eni's change:
![16-44-b9hv1-lfw6f](https://user-images.githubusercontent.com/127662/212728963-6afe2164-5c61-45fa-a90e-face2cbced60.gif)

It shows you where can you edit the text.

Reply:
gotcha 👍

Reply:
With this fix inside the text area the chrome text cursor is used automatically.
I think the insert cursor doesn't fit here because clicking outside of the text area just finishes the text editing but doesn't allow inserting?